### PR TITLE
[PL-38837] Add rotate tokens content to secure delegates topic

### PR DESCRIPTION
**Pull request by brian-f-harness**
Add rotate delegate tokens content to secure delegates with tokens topic

For reviewers, [a preview](https://6464f3e4ab554f02f0ce1cee--harness-developer.netlify.app/docs/platform/Delegates/secure-delegates/secure-delegates-with-tokens) is available.

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — COMMENTED** [review]
Suggested changes

**Review by pratmit — DISMISSED** [review]
LGTM

File: docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md [hunk 1]
```diff
@@ -1,135 +1,184 @@
...
 helpdocs_category_id: m9iau0y3hv
 helpdocs_is_private: false
 helpdocs_is_published: true
 ---
 
-Delegate tokens are used by Harness to encrypt communication between Harness Delegates and the Harness Manager. By default, when a new Harness account is created, all Harness Delegates in that account include the same token.
+Harness uses delegate tokens to encrypt communication between Harness Delegates and the Harness Manager. By default, when a new Harness account is created, all Harness Delegates in that account include the same token.
 
-You can further secure Delegate to Harness communication by replacing the default Delegate token with new tokens. You can add and revoke Delegate tokens per your governance policies and replace revoked tokens with custom tokens when needed.
+You can further secure delegate to Harness communication by replacing the default delegate token with new tokens. You can add and revoke delegate tokens per your governance policies and replace revoked tokens with custom tokens when needed. You can also rotate your tokens and store them in a secret manager.
```
Comment (douglas-j-bothwell) [inline docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md:13]:
```suggestion
You can further secure delegate-to-Harness communication by replacing the default delegate token with new tokens. You can add and revoke delegate tokens per your governance policies and replace revoked tokens with custom tokens when needed. You can also rotate your tokens and store them in a secret manager.
```

File: docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md [hunk 2]
```diff
@@ -1,135 +1,184 @@
...
-    image: harness/delegate:23.01.78102  
-    environment:  
-      - ACCOUNT_ID=12345678910  
-      - ACCOUNT_SECRET=[enter new token here]  
-      - MANAGER_HOST_AND_PORT=https://app.harness.io  
-      - WATCHER_STORAGE_URL=https://app.harness.io/public/pro  
-...
-```
-Create a new container: `docker-compose -f docker-compose.yaml up -d`.
+You can rotate and store your delegate tokens in 3P secret manager and reference them as needed.
```
Comment (douglas-j-bothwell) [inline docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md:154]: Is 3P a known term? Google says No results found for "3P secret manager".

**Review by douglas-j-bothwell — DISMISSED** [review]

File: docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md (hunk 2 above)
Comment (brian-f-harness) [reply docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md:154]: @douglas-j-bothwell good question. I'll check with Prateek.

**Pull request merged**